gamemode/config/rafts.lua

```diff
@@ -28,3 +28,21 @@ Config.BUILDING_REQUIREMENTS = {
         nail = 10
     }
 }
+
+Config.RAFT_VERTICAL_OFFSET = 10
+
+-- Used for spawning rafts
+-- Currently set up for rvr_water, this should be set to the minimum corner in x and y
+Config.MAP_MIN = Vector( -15000, -15000 )
```

Comment:
should these be map sepcific 
Config.Map["rvr_water"].MAP_MIN
and
Config.Map["rvr_water"].MAP_MAX